buildlib/azure-pipelines.yml

```diff
@@ -10,33 +10,65 @@ resources:
     - container: centos7
       image: ucfconsort.azurecr.io/ucx/centos7:1
       endpoint: ucfconsort_registry
+    - container: fedora
+      image: ucfconsort.azurecr.io/ucx/fedora:30
+      endpoint: ucfconsort_registry
 
 stages:
   - stage: Build
     jobs:
-      - job: Compile
-        displayName: Compile Tests
-        pool:
-          vmImage: 'Ubuntu-16.04'
+      - job: latest_cc
+        displayName: Latest CCs and CppCheck
+        container: fedora
         steps:
-          - bash: |
-              ./autogen.sh
+          - bash: ./autogen.sh
             displayName: Setup autotools
 
           - bash: |
-              mkdir build && cd build
-              ../configure --disable-numa
-            displayName: Configure
+              mkdir build-gcc && cd build-gcc
```

Comment:
You should have set -e if using command sequences, IIRC. In all of the bash stanzas

Reply:
Adde to a long one, don't think it is required in every bash script.

Reply:
You need it any time there is a pipeline, ie the | tee below, possibly also for the `` expressions too. Generally I find it simpler just to include it than to try to keep track if the script has one of the required pipeline, list, or compound patterns where it matters.